rc/lsp.kak

```diff
@@ -1944,3 +1956,12 @@ define-command lsp-goto-next-match -docstring 'DEPRECATED: use lsp-next-location
 define-command lsp-goto-previous-match -docstring 'DEPRECATED: use lsp-previous-location. Jump to the previous goto match' %{
     lsp-previous-location '*goto*'
 }
+
+declare-option -hidden str lsp_hover_fifo
+
+set-option global lsp_hover_fifo %sh{
+    tmpdir=$(mktemp -q -d -t 'lsp-hover-client.XXXXXX' 2>/dev/null || mktemp -q -d)
+    output="$tmpdir/fifo"
+    mkfifo "$output"
```

Comment:
We should probably create this lazily (the first time the user requests the hover-in-in-hoverclient)

Also we should clean up the tempdir. Maybe in `lsp-exit-editor-session`/`lsp-stop` which are run on our KakEnd hook -- they also run `remove-hooks global lsp`.

Other commands like `lsp-replace-selection` clean up their temp files immediately.
Though the persistent FIFO seems fine.